runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -281,34 +281,15 @@ ClientSessionData::printStats()
    }
 
 ClientSessionData::ClassInfo::ClassInfo() :
-   _romClass(NULL),
```

Comment:
I think that we need to keep these initializations. As as a principle, a constructor should initialize all fields or none. Maybe order them so that all caches stay together.

Reply:
Thought these were removed in the `jitaas` branch. I'll back port the change to `jitaas` branch in #7496.